### ISSUE TITLE
View/add/search contacts from web interface

### ISSUE THREAD

**Issue by grundic** [opensourcehacker/sevabot#19]
Add functionality to view/add/search contacts from web page.

**Comment by miohtama** [comment]
Good idea. What kind of use cases you have for this? As the contact list of bot will be pretty limited unless every person friends with the bot.

**Comment by grundic** [comment]
Actually, viewing is already implmented.
Add/search is not a must have feature, just cool addition. For example, it administrator would not like to give full access to host (ssh, vnc), some task could be done via web via other people.
